molecule/util.py

```diff
@@ -126,6 +129,39 @@ def write_template(src, dest, kwargs={}, _module='molecule', _dir='template'):
         f.write(template.render(**kwargs))
 
 
+def process_templates(template_dir, extra_context, output_dir, overwrite=True):
+    """
+    Process templates as found in the named directory.
+
+    :param template_dir: An absolute or relative path to a directory where the templates are located. If the provided
+                         directory is a relative path, it is resolved using a known location.
+    :type template_dir: str
+    :param extra_context: A set of values that are used to override default or user specified values.
+    :type extra_context: dict or None
+    :param output_dir: An absolute path to a directory where the templates should be written to.
+    :type output_dir: str
+    :param overwrite: Whether or not to overwrite existing templates. Defaults to true.
+    :type overwrite: bool
```

Comment:
Sure. I saw that you were allowing a max line length of 120 in the flake8 config so I was just going off that. I am happy to use 80.